src/auro-accordion.js

```diff
@@ -100,24 +135,44 @@ export class AuroAccordion extends LitElement {
     content.style.height = `${height}px`;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has('iconRight')) {
+      this.requestUpdate();
+    }
+  }
+
   // function that renders the HTML and CSS into  the scope of the component
   render() {
 
-    const chevronHtml = this.chevron
-      ? html`
-          <div class="iconWrapper" part="chevron">
-            ${this.generateIconHtml(this.expanded ? chevronUp.svg : chevronDown.svg)}
-          </div>`
-      : html``
+    const buttonClasses = {
+      "trigger": true,
+      "iconRight": this.iconRight
+    }
+
+    const chevronHtml = this.noChevron
+      ? html``
+      : html`
+        <div class="iconWrapper" part="chevron" slot="icon">
+          ${this.generateIconHtml(this.expanded ? chevronUp.svg : chevronDown.svg)}
+        </div>
+      `;
 
     return html`
-      <div class="componentWrapper">
-        <button class="trigger" id="accordionTrigger" aria-controls="accordionContent" aria-expanded="${this.expanded}" @click="${this.toggle}" part="trigger">
-          <slot name="trigger"></slot>
+      <div class="componentWrapper" part="accordion">
+        <auro-accordionbutton 
+          ?fluid="${this.fluid || this.emphasis}" 
```

Comment:
This attribute is needed because the HTML5 button within `auro-accordionbutton` does not have a width of 100% if the `auro-accordionbutton` does not contain the `fluid` attribute. This can be seen specifically with the `emphasis` example, as the chevron does not appear on the right side of the component without the attribute.

Reply:
Default accordion. No `fluid` use. Using `display: flex` 

<img width="1507" alt="Screen Shot 2023-10-05 at 12 46 26 PM" src="https://github.com/AlaskaAirlines/auro-accordion/assets/181089/b4c7fddb-0665-4898-8ed4-202ddec848d2">

My point is that in the API you state `fluid` as `If set, the trigger and content will be 100% width.` But this is TRUE without that being set. 

If you need to use this to pass this to the width of the button, then that needs to be more clear. Also, why not just have `fluid` by default? The width of the button is not really necessary as the whole trigger is clickable. This is only not the case when you are using the current `alignRight` attribute.

Reply:
I have updated the code and removed the `fluid` attribute from the API. There is still an if statement within `auro-accordionbutton` that applies the `fluid` attribute to the button when the `emphasis` attribute is present. I am not applying `fluid` by default because it affects single accordions that have a chevron appearing on the right side by pushing the chevron to the edge of the container, when it should render next to the label.